org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -197,11 +197,11 @@ public static Object get(Object model, String name) throws IllegalAccessExceptio
 		}
 		return null;
 	}
-	
+
 	private static Object get(Object model, Class<?> clazz, String name) throws IllegalAccessException, IllegalArgumentException, InvocationTargetException {
-		
+
 		if (clazz==null || clazz.equals(Object.class)) return null;
-		
+
 		final String getter = getGetterName(name).toLowerCase();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getGetterName" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)